docs/data/joy/getting-started/tutorial/LoginFinal.js

```diff
@@ -23,17 +24,13 @@ const ModeToggle = () => {
     <Button
       variant="outlined"
       onClick={() => {
-        if (mode === 'light') {
-          setMode('dark');
-        } else {
-          setMode('light');
-        }
+        setMode(mode === 'light' ? 'dark' : 'light');
```

Comment:
It seems mentally simpler to follow by calling `setMode` once rather than branching it entirely.